crates/core_arch/src/arm/neon.rs

```diff
@@ -1440,12 +1445,131 @@ pub unsafe fn vtbx4_p8(a: poly8x8_t, b: poly8x8x4_t, c: uint8x8_t) -> poly8x8_t
     ))
 }
 
+// uint8_t vget_lane_u8 (uint8x8_t v, const int lane)
```

Comment:
Those do not translate properly and I'm at a loss how to get it working. It would be great to get some advice on how to get that working.

godbolt fails with all of the vget_lane commands and I couldn't find any implementation of it anywhere either.

Reply:
Do you have an example that fails in godbolt ? Probably one just needs to make `lane` a compile time constant.

Reply:
I just followed the godbolt example in #148 to test the intrinsics it doesn’t resolve for either aarch or arm

Reply:
https://godbolt.org/z/tsjFBs to be precise

Reply:
This might help: https://godbolt.org/z/fN88tH

Reply:
I did try that approach, it doesn't lead to llvm turning it into a `umov` instruction. `UMOV Rd,Vn.B[lane]`) is what we should see based on the manual both the extract and the match compile down to `ldrb` instead

Reply:
I'd prefer to implement this nicely but I can skip the vget_* or implement them with extract for later investigation - personally I'm a stumped

Reply:
for record keeping, this is the closest I found: https://github.com/llvm-mirror/llvm/commit/c52cd839b99cd7577ae871c46d478c1957972c03#diff-aa6ebb35c8cd5bffded9460741f35d2dR11

Reply:
Pulled this out of the PR